src/families/ethereum/EditFeeUnitEthereum.js

```diff
@@ -0,0 +1,202 @@
+// @flow
+import React, { useMemo, useState, useCallback } from "react";
+import { View, StyleSheet } from "react-native";
+import { translate } from "react-i18next";
+import Slider from "react-native-slider";
+import type { NavigationScreenProp } from "react-navigation";
+import type { Account, TokenAccount } from "@ledgerhq/live-common/lib/types";
+import type { Transaction } from "@ledgerhq/live-common/lib/bridge/EthereumJSBridge";
+import {
+  inferDynamicRange,
+  reverseRangeIndex,
+  projectRangeIndex,
+} from "@ledgerhq/live-common/lib/range";
+import { getAccountBridge } from "@ledgerhq/live-common/lib/bridge";
+import { getMainAccount } from "@ledgerhq/live-common/lib/account";
+import { BigNumber } from "bignumber.js";
+
+import { editTxFeeByFamily } from "../helpers";
+import type { T } from "../../types/common";
+import colors from "../../colors";
+import LText from "../../components/LText";
+import CurrencyUnitValue from "../../components/CurrencyUnitValue";
+import SettingsRow from "../../components/SettingsRow";
+import Button from "../../components/Button";
+
+type Props = {
+  account: Account | TokenAccount,
+  parentAccount: ?Account,
+  transaction: Transaction,
+  t: T,
+  navigation: NavigationScreenProp<*>,
+};
+
+const GasSlider = React.memo(({ defaultGas, value, onChange }: *) => {
+  const range = useMemo(() => inferDynamicRange(defaultGas), [defaultGas]);
+  const index = reverseRangeIndex(range, value);
+  const setValueIndex = useCallback(
+    i => onChange(projectRangeIndex(range, i)),
+    [range, onChange],
+  );
+
+  return (
+    <Slider
+      value={index}
+      step={1}
+      onValueChange={setValueIndex}
+      minimumValue={0}
+      maximumValue={range.steps - 1}
+      thumbTintColor={colors.live}
+      minimumTrackTintColor={colors.live}
+    />
+  );
+});
+
+const fallbackGasPrice = BigNumber(10000000000);
+
+const EditFeeUnitEthereum = ({
+  account,
+  parentAccount,
+  transaction,
+  t,
+  navigation,
+}: Props) => {
+  const mainAccount = getMainAccount(account, parentAccount);
+  const bridge = getAccountBridge(account, parentAccount);
+
+  const [gasPrice, setGasPrice] = useState(() =>
+    bridge.getTransactionExtra(mainAccount, transaction, "gasPrice"),
+  );
+  const feeCustomUnit = bridge.getTransactionExtra(
+    mainAccount,
+    transaction,
+    "feeCustomUnit",
+  );
+
+  const fees = bridge.getTransactionNetworkInfo(mainAccount, transaction);
```

Comment:
this is guaranteed to be filled in mobile paradigm. You can assume it's set. so feel free to `if (!fees) return null` after this line so you don't need a `fallbackGasPrice`